openbb_platform/core/tests/app/logs/test_logging_service.py

```diff
@@ -153,7 +153,7 @@ class MockCredentials(BaseModel):
             "mock_route",
```

Comment:
I made a few changes in this commit to prevent the same bug to happen again -> https://github.com/OpenBB-finance/OpenBBTerminal/pull/6548/commits/79eefc8a274facb6603b0dd4058afdbc0b216159